tests/test_filter.py

```diff
@@ -144,6 +171,7 @@ def grid_type_and_input_ds(request):
 def test_filter(grid_type_and_input_ds, filter_args):
     grid_type, da, grid_vars = grid_type_and_input_ds
     filter = Filter(grid_type=grid_type, grid_vars=grid_vars, **filter_args)
+    filter.plot_shape()
```

Comment:
To me, this is a good enough "smoke test" for now. We call the function and make sure it doesn't error.